src/jit/compiler.h

```diff
@@ -1045,9 +1121,15 @@ class JitTimer
     // Ends the current phase (argument is for a redundant check).
     void EndPhase(Phases phase);
 
+#if MEASURE_CLRAPI_CALLS
+    // Start and end a timed CLR API call.
+    void CLRapiCallEnter(unsigned apix);
+    void CLRapiCallLeave(unsigned apix);
+#endif // MEASURE_CLRAPI_CALLS
```

Comment:
In general, if we're using CamelCase (which we are...) then it seems the first letter of a word/name needs to be caps even if it follows an all-caps thing. Make, make the all-caps thing mixed, like is done in the C# standard. E.g., ClrApiCallEnter()